team.go

```diff
@@ -79,18 +79,31 @@ func (c *Client) Team(id int64) (*Team, error) {
 // AddTeam makes a new team
 // email arg is an optional value.
 // If you don't want to set email, please set "" (empty string).
-func (c *Client) AddTeam(name string, email string) error {
+// When team creation is successful, returns the team id
```

Comment:
I think it's best to standardize on ID as short for identity:
```suggestion
// When team creation is successful, returns the team ID.
```